.github/workflows/check-new-package.yml

```diff
@@ -0,0 +1,94 @@
+name: Check New Theia Package
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronized, reopened]
+    paths:
+      - 'packages/*/package.json'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  detect-new-packages:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
+
+      - name: Detect if a new package was added
+        id: detect
+        run: |
+          git fetch origin ${{ github.base_ref }}
+          NEW_PACKAGES=$(git diff --name-only --diff-filter=A origin/${{ github.base_ref }}...HEAD | grep -E '^packages/[^/]+/package\.json$' || true)
+
+          if [ -z "$NEW_PACKAGES" ]; then
+            echo "new_package_found=false" >> $GITHUB_OUTPUT
+            echo "No new packages detected"
+          else
+            echo "new_package_found=true" >> $GITHUB_OUTPUT
+            PACKAGE_NAMES=$(echo "$NEW_PACKAGES" | sed 's|packages/\([^/]*\)/package\.json|\1|' | tr '\n' ',' | sed 's/,$//')
+            echo "package_names=$PACKAGE_NAMES" >> $GITHUB_OUTPUT
+            echo "New packages detected: $PACKAGE_NAMES"
+          fi
+
+      - name: Post or update checklist comment
+        if: steps.detect.outputs.new_package_found == 'true'
+        uses: actions/github-script@ffc2c79a5b2490bd33e0a41c1de74b877714d736 # v3.2.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const packageNames = '${{ steps.detect.outputs.package_names }}';
+            const packageList = packageNames.split(',').map(name => `\`${name}\``).join(', ');
+
+            const commentBody = "### New `@theia` Package(s) Detected
+            
+            This PR adds the following new package(s): ${packageList}
+
+            Please ensure the following checklist items are completed before merging:
+
+            - [ ] `package.json` contains all required fields (name, version, description, license, etc.) and scripts
+            - [ ] `README.md` was added (please align with the existing README structure, e.g. [README.md](https://github.com/eclipse-theia/theia/blob/master/packages/editor/README.md))
+            - [ ] Config files (`tsconfig.json`, `.eslintrc.js`) were added and align with the existing packages
+            - [ ] Folder structure follows Theia conventions (see [Code Organization](https://github.com/eclipse-theia/theia/blob/master/doc/code-organization.md))
+            - [ ] Package is added to the example applications (i.e. `browser`, `browser-only`, `electron`)
+            - [ ] New packages must be published manually by a Theia committer initially.
+                  Please open a follow-up ticket to inform the release team that the new package needs to be published initially
+                  (see also [Release Process](https://github.com/eclipse-theia/theia/blob/master/doc/Publishing.md#2-release-process))
```

Comment:
IMHO we should set a tag on these issues so we can  then also mention the tag in the release guide, so its easier to find those issue during the release. WDYT?